### DISCUSSION TITLE
OpenMage Logo

### DISCUSSION THREAD

**Discussion by addison74** [OpenMage/magento-lts#2236] [General]
Please take a look to this website logo https://www.mynewsdesk.com/wp-content/uploads/2022/01/cropped-favicon-192x192.png. It seems to be the OpenMage logo in which letter "m" is on the opposite side. If they have a trademark it could be a problem. Maybe we should have a logo with OM letters and not "m" which is coming from Magento.

**Comment by sreichel** [comment]
PR?
